lib/wsdl.js

```diff
@@ -1775,15 +1865,15 @@ WSDL.prototype.objectToXML = function(obj, name, nsPrefix, nsURI, isFirst, xmlns
         }
       }
 
-      if (!Array.isArray(child)) {
+            //if (!Array.isArray(child)) {
```

Comment:
Please don't just comment out "unused" loc, remove them if they are not needed anymore